PhotoStream/app/src/main/java/com/keenanzucker/photostream/ImageContract.java

```diff
@@ -0,0 +1,19 @@
+package com.keenanzucker.photostream;
+
+import android.provider.BaseColumns;
+
+/**
+ * Created by keenan on 10/2/15.
+ * Image Contract class implements BaseColumns and stores database table variables.
+ */
+public class ImageContract {
+
+    public ImageContract(){}
+
+    public static abstract class FeedEntry implements BaseColumns {
```

Comment:
http://programmers.stackexchange.com/questions/81499/when-should-you-use-a-private-inner-class
I know the tutorial has this done in this way, so its cool, but I just want you to think about if it makes any sense that it is done this way @keenanzucker . I personally disagree with this, but I would like to know if you have any thoughts about it. If not that's okay.